### PR TITLE
Signup screen added

### PR DESCRIPTION
**Pull request by Monte9**
Feel free to leave comments.

### CODE REVIEW

File: screens/signup/index.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    flexDirection: 'column',
+    flex: 1,
+    backgroundColor: 'transparent'
+  },
+  bg: {
+    position: 'absolute',
+    left: 0,
+    top: 0,
+    width: Dimensions.get('window').width,
+    height: Dimensions.get('window').height
```
Comment (browniefed) [inline screens/signup/index.js]: This is part has been revamped in latest code, Checkout the new code structure of the login page

File: screens/signup/index.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  headerIconView: {
+    flex: .1,
```
Comment (browniefed) [inline screens/signup/index.js]: Should probably never do point flex values. This makes things complicated

File: screens/signup/index.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    top: 35,
+    left: 15,
+  },
+  backButtonIcon: {
+    resizeMode: 'contain',
+    width: 25,
+    height: 25
+  },
+  headerTitleView: {
+    flex: 0.1,
```
Comment (browniefed) [inline screens/signup/index.js]: Same here, just remove flex property.

File: screens/signup/index.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+    paddingBottom: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    alignSelf: 'center',
+    fontSize: 15,
+  },
+  inputs: {
+    paddingTop: 20,
+    paddingBottom: 10,
+    flex: .40
```
Comment (browniefed) [inline screens/signup/index.js]: Remove flex here.

File: screens/signup/index.js [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+  },
+  footerContainer: {
+    flex: 0.15,
```
Comment (browniefed) [inline screens/signup/index.js]: Remove flex.

File: screens/signup/index.js [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  signupText: {
+    color: '#FFF',
+  },
+  signin: {
+    justifyContent: 'center',
+    alignItems: 'center',
+    flex: 0.05
```
Comment (browniefed) [inline screens/signup/index.js]: Remove flex.

**Comment by browniefed** [comment]
Needs a little work on the flex.
Checkout the `Login` page I updated all the content in it.

**Comment by browniefed** [comment]
More explanation on the flex. Most of those things probably just have defined heights, we can just let them be at their normal heights. The % flex stuff was a good way to basically do % widths, just can be confusing to new comers trying to learn flex.

**Comment by Monte9** [comment]
I addressed the PR comments. And thanks for the explanation on flex, but I disagree on not using flex. I see what you mean by using a fixed height and it is so, but I am using flex to place the elements where they need to be on the screen. Here are some screenshots showing the view on various phones using flex:
screen shot 2016-11-22 at 10 13 14 pm
screen shot 2016-11-22 at 10 13 14 pm
screen shot 2016-11-22 at 10 13 14 pm

And here's the screeshot without flex:
screen shot 2016-11-22 at 10 13 14 pm

**Comment by browniefed** [comment]
Ah sorry, I didn't mean don't use flex, I just meant don't use flex with hard to determine ratios.

**Comment by Monte9** [comment]
Ah that makes more sense. I updated the flex values to whole numbers.

**Comment by browniefed** [comment]
Thanks, I'll get this merged likely tomorrow, and then fix things up to add in some navigation or something to swipe between screens.

**Comment by Monte9** [comment]
Sounds good. What do you plan to use for navigation? NavigationExperimental?

**Comment by browniefed** [comment]
Likely ExNavigation. Easier to deal with.

**Comment by browniefed** [comment]
I've cleaned up the styling if you care to look at the code now.

**Pull request merged**